packages/cli/import_contract_types_from_sdk.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
```

Comment:
Where is this file being used?

Reply:
good catch. it was removed by another PR and got re-added when merging